Task4.1/Task4.1.c

```diff
@@ -0,0 +1,248 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+
+/**
+* @brief варианты заполнения массива числовыми значениями
+*/
+enum mass_input_choise {
+	/**
+	* @brief ручным способом
+	*/
+	arms = 1,
+	/**
+	* @brief заполнение случайными числами
+	*/
+	random = 2
+};
+
+/**
+* @brief проверяет вводимое значение (является ли числом)
+* @return возвращает введённое значение, если оно является числом
+*/
+int input(const char* message);
+
+/**
+* @brief Функция для печати массива.
+* @param mass - массив.
+* @param size - длинна массива.
+* @return Массив.
+*/
+void print_mass(int* mass, const size_t size);
```

Comment:
Если параметры функции не изменяются, то они явно должны быть обозначены `const`
[By Alex Allain](https://www.cprogramming.com/tutorial/const_correctness.html)
>The const keyword allows you to specify whether or not a variable is modifiable. You can use const to prevent modifications to variables and const pointers and const references prevent changing the data pointed to (or referenced).
>Const дает вам возможность более четко документировать вашу программу и фактически обеспечивать соблюдение этой документации. Ключевое слово const обеспечивает вашим пользователям гарантии, позволяющие оптимизировать производительность без угрозы повреждения их данных, обеспечивая соблюдение вашей документации. Например, константные ссылки позволяют указать, что упомянутые данные не будут изменены; это означает, что вы можете использовать константные ссылки как простой и немедленный способ повышения производительности для любой функции, которая в настоящее время принимает объекты по значению, не беспокоясь о том, что ваша функция может изменить данные. Даже если это произойдет, компилятор предотвратит компиляцию кода и предупредит вас о проблеме. С другой стороны, если бы вы не использовали константные ссылки, у вас не было бы простого способа гарантировать, что ваши данные не были изменены.